git-subhistory.sh

```diff
@@ -16,10 +16,11 @@ test $# = 0 && set -- -h
 
 OPTS_SPEC="\
 git-subhistory split <subproj-path> (-b | -B) <subproj-branch>
-git-subhistory merge <subproj-path> <subproj-branch>
+git-subhistory merge <subproj-path> (-a) <subproj-branch>
 --
 q,quiet         be quiet
 v,verbose       be verbose
+a,auto          use auto-generated commit message without prompting user
```

Comment:
Hmmm, this feature makes sense, but it should be called `--no-edit` to match the `git-merge` option:

> The `--no-edit option` can be used to accept the auto-generated message

https://git-scm.com/docs/git-merge

In fact, if there's a good way to take all extraneous options that `git-subhistory` doesn't care about and pass them to `git-merge`, that'd be dope. But handcoding a few of the most useful ones is fine for now.